promgen/static/js/promgen.vue.js

```diff
@@ -323,3 +351,95 @@ app.component('exporter-test', {
         }
     }
 });
+
+app.component('silence-table-modal', {
+    template: '#silence-table-modal-template',
+    delimiters: ['[[', ']]'],
+    mixins: [mixins],
+    data() {
+        return {
+            state: silenceTableStore.state,
+            form: {
+                labels: [],
+                label: '',
+                value: ''
+            },
+            store: dataStore
+        };
+    },
+    computed: {
+        globalMessages() {
+            return globalStore.state.messages;
+        },
+        activeSilences() {
+            return this.$root.activeSilences || [];
+        },
```

Comment:
Since we can get a list from activeSilences() it seems like we could do a quick filter on that, to provide a list of suggestions for the matcher label field, and then do something similar for the matcher value based on which label is selected 🤔

Reply:
ah, I see. I will look into it.